paimon-flink/paimon-flink-cdc/src/main/java/org/apache/paimon/flink/sink/cdc/CdcDynamicTableParsingProcessFunction.java

```diff
@@ -100,7 +100,10 @@ public void processElement(T raw, Context context, Collector<Void> collector) th
                             try {
                                 catalog.createTable(identifier, schema, true);
                             } catch (Exception e) {
-                                LOG.error("create newly added paimon table error.", e);
+                                LOG.error(
+                                        "Cannot create newly added Paimon table "
+                                                + identifier.getFullName(),
+                                        e);
```

Comment:
LOG.error("Cannot create newly added Paimon table: {}.", identifier.getFullName(), e);  ?

Reply:
Seems this method will consider the exception as the second string argument? 
```
@param format the format string
@param arg1   the first argument
@param arg2   the second argument
public void error(String format, Object arg1, Object arg2);
```

Reply:
slf4j will internally judge the last parameter.

![image](https://github.com/apache/incubator-paimon/assets/37063904/8b251fd0-dd3f-4ea4-b6b5-512086ed02f8)

https://github.com/qos-ch/slf4j/blob/5ff6f2c385c36ea4f8b85cacd69f7ca891c37818/slf4j-api/src/main/java/org/slf4j/helpers/AbstractLogger.java#L389-L395

Reply:
That makes sense.